pkg/hooks/loader.go

```diff
@@ -175,6 +194,7 @@ func (h *Hook) SetTcsMocks(m []*models.Mock) {
 func (h *Hook) SetConfigMocks(m []*models.Mock) {
 	h.configMocks.deleteAll()
 	for index, mock := range m {
+		h.UpdatePersistentMatchedMock(mock.Name, false)
 		mock.TestModeInfo.SortOrder = index
```

Comment:
since we're keep only the mocks that are used better to remove this?

Reply:
doing this would mean no config mocks in persistentMatchedMocks and only tcsMocks will get written in mocks.yaml